net-dns/knot/knot-2.5.3-r1.ebuild

```diff
@@ -36,14 +48,21 @@ DEPEND="${RDEPEND}
 S="${WORKDIR}/${P/_/-}"
 
 src_configure() {
+	local my_conf=()
+	for u in "${KNOT_MODULES[@]//+/}"; do
```

Comment:
Actually, `"${KNOT_MODULES[@]#+}"` would be more precise here. I'll change it for you though.